### PR TITLE
Update refs to Java 8 to Java 11 as min version

### PR DESCRIPTION
**Pull request by Tr3yb0**

### CODE REVIEW

File: src/content/0.15.0/docs/start/anywhere.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ There are two options for running Fluree:
 
 ### Download
 
-To run Fluree, you need Java Server 8 (Java 1.8) or above. To verify your version of Java, you can type `java -version` in the terminal.
+To run Fluree, you need Java 11 or above. To verify your version of Java, you can type `java -version` in the terminal.
```
Comment (ldw1007) [inline src/content/0.15.0/docs/start/anywhere.md:10]: Actually, v0.15 & v0.16 were tested/running under Java 8.  
You need to create a copy of this file for v1.0

File: src/content/0.15.0/docs/start/anywhere.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ To change the location of the webserver (setting name `fdb-api-port`), or any ot
 
 #### Troubleshooting
 
-If the above message is not displaying in your terminal, the terminal should print out a relevant error message. Common errors include your chosen port already being in use and not having Java 8 or above installed. 
+If the above message is not displaying in your terminal, the terminal should print out a relevant error message. Common errors include your chosen port already being in use and not having Java 11 or above installed.
```
Comment (ldw1007) [inline src/content/0.15.0/docs/start/anywhere.md:52]: same comment as above - changes need to be in a new file for v1.0

File: src/content/0.15.0/docs/start/anywhere.md [hunk 3]
```diff
@@ -49,9 +49,9 @@ To change the location of the webserver (setting name `fdb-api-port`), or any ot
 
 #### Troubleshooting
 
-If the above message is not displaying in your terminal, the terminal should print out a relevant error message. Common errors include your chosen port already being in use and not having Java 8 or above installed. 
+If the above message is not displaying in your terminal, the terminal should print out a relevant error message. Common errors include your chosen port already being in use and not having Java 11 or above installed. 
 
-- If you see an error "missing 'server' JVM", you need to install Java Server JRE. See <a href="https://docs.oracle.com/en/java/" target="_blank">Oracle documentation</a> to select the appropriate platform, version (e.g., 8) and operating system.
+- If you see an error "missing 'server' JVM", you need to install Java Server JRE. See <a href="https://docs.oracle.com/en/java/" target="_blank">Oracle documentation</a> to select the appropriate platform, version (e.g., 11) and operating system.
```
Comment (ldw1007) [inline src/content/0.15.0/docs/start/anywhere.md:54]: :) same comment.

**Review by ldw1007 — CHANGES_REQUESTED** [review]
Let me know if you need help creating a new "anywhere.md" file for v1.0

**Review by Tr3yb0 — COMMENTED** [review]
corrected issues

File: src/content/0.15.0/docs/start/anywhere.md (hunk 2 above)
Comment (ldw1007) [reply src/content/0.15.0/docs/start/anywhere.md:52]: actually, this needs to be changed to Java 11

**Review by ldw1007 — APPROVED** [review]

**Pull request merged**